pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -546,6 +546,9 @@ const (
 	AnnotationNodeLocalDNSForceTcpToClusterDns = "alpha.featuregates.shoot.gardener.cloud/node-local-dns-force-tcp-to-cluster-dns"
 	// AnnotationNodeLocalDNSForceTcpToUpstreamDns enforces upgrade to tcp connections for communication between node local and upstream dns.
 	AnnotationNodeLocalDNSForceTcpToUpstreamDns = "alpha.featuregates.shoot.gardener.cloud/node-local-dns-force-tcp-to-upstream-dns"
+	// AnnotationManagedResourceSecretChecksum is a key for an annotation on a Managed Resource containing checksum for the secrets
+	// referenced by managed resource.
+	AnnotationManagedResourceSecretChecksum = "checksum/secret-data"
```

Comment:
This is no longer needed now?

Reply:
Ah, I missed removing it in an earlier commit. Thanks.